src/locales/translations/it.ts

```diff
@@ -94,4 +94,7 @@ export default {
   run_as_admin: "Esegui come amministratore",
   settings_general_tab_title: "Generale",
   settings_lang_tab_title: "Lingua",
+  settings_advanced_tab_title: "Avanzate",
+  settings_advanced_discord_status: "Mostra stato su Discord",
+  join_discord: "Unisciti al Server Discord",  

```

Comment:
Is there a way to make this one smaller?

Reply:
Are you referring just to the "join_discord" one?

Reply:
Yes,
also there are two more added now in master, would appreciate it if you translate those as well